### DISCUSSION TITLE
Github Pages 404

### DISCUSSION THREAD

**Discussion by nataliemiller03** [community/community#136027] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

👋 Thanks in advance if anyone has the time to look at this!

I've been trying to deploy my site [from this repo to github pages](https://github.com/nataliemiller03/nataliemiller03.github.io) and then to a custom domain name. It looks like the redirect works just fine (nataliemiller03.github.io takes you to nataliecmiller.com) however I'm consistently getting a 404 error from github pages. I've set it up to point to the gh-pages branch which contains an index.html file. 

I'm not sure what's going on as I've deployed sites using this method in the past. Any help would be appreciated!

**Comment by ebndev** [comment]
Hi @nataliemiller03, thanks for posting in GitHub Discussions 😄 

I've gone ahead and moved this post to our `Pages` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction!

**Comment by LiteBrite82** [comment]
Hi @nataliemiller03 
I don't see any 404s on my end. can you send a screenshot?
<img width="1719" alt="Screenshot 2024-08-16 at 3 08 57 PM" src="https://github.com/user-attachments/assets/bf0ccfe0-bd1b-4fec-8c0e-128285dfdb2b">

**Reply by LiteBrite82** [reply]
I like how you handle page change BTW; Nice!

**Reply by davevad93** [reply]
Hi @nataliemiller03 , as @LiteBrite82 said, I have checked your website and there are no 404 errors at all.

**Reply by LiteBrite82** [reply]
P.s. the default branch should still be `main` 
Set your deployment branch from `Pages Settings`:
<img width="1704" alt="Screenshot 2024-08-16 at 3 12 51 PM" src="https://github.com/user-attachments/assets/3bf6e647-1eb1-462d-bd45-802d37fd51a4">

**Reply by nataliemiller03** [reply, marked as answer]
Thanks y'all I did get it working! Chrome was taking way too long to bust the cache on my end

**Reply by davevad93** [reply]
You're welcome @nataliemiller03 , glad it's working!
